interface/usergroup/user_admin.php

```diff
@@ -388,7 +388,7 @@ function authorized_clicked() {
 <tr>
   <?php
     if($iter["picture_url"]){
-      echo '<td style="width:180px;"><img id="prof_img" src="../../profile_pictures/'.$iter["picture_url"].'"';
+      echo '<td style="width:180px;"><img id="prof_img" src="../../sites/'.$_SESSION['site_id'].'/profile_pictures/'.$iter["picture_url"].'"';
```

Comment:
relpath

Reply:
@aethelwulffe  the `src` attribute of images does  not recognise `relpath(...)`. It must be from the root of the webserver doc like `localhost/lh-ehr/mua.jpeg` for it to function.